can-map.js

```diff
@@ -22,19 +22,18 @@ var Construct = require('can-construct');
 var ObservationRecorder = require('can-observation-recorder');
 var ObserveReader = require('can-stache-key');
 var canCompute = require('can-compute');
-var singleReference = require('can-util/js/single-reference/single-reference');
+var singleReference = require('can-single-reference');
 
 var namespace = require("can-namespace");
-var dev = require("can-util/js/dev/dev");
+var dev = require("can-log/dev/dev");
 var CID = require("can-cid");
-var deepAssign = require("can-util/js/deep-assign/deep-assign");
 //var isFunction = require("can-util/js/is-function/is-function");
-var assign = require("can-util/js/assign/assign");
+var assign = require("can-assign");
```

Comment:
It worries me a little bit that the tests are passing but this module isn’t in package.json

Reply:
Thanks, fixed!